webroot/js/sentences_lists.menu.js

```diff
@@ -52,6 +52,7 @@ $(document).ready(function() {
         // directly after the selection in the <select>.
         $(".validateButton").off();
         $(".validateButton").click(function(){
+            sentenceId = $(this).parent().attr("id").replace("addToList", "");

```

Comment:
A way of getting the sentence ID that feels less of a workaround would be to set the `data-sentence-id` attribute on the `.validateButton` element instead of the `.addToList` element.

This way you can use `sentenceId = $(this).attr("data-sentence-id")` here, and you can remove it from line 45.

Reply:
> A way of getting the sentence ID that feels less of a workaround would be to set the `data-sentence-id` attribute on the `.validateButton` element instead of the `.addToList` element.

Done.

> This way you can use `sentenceId = $(this).attr("data-sentence-id")` here, and you can remove it from line 45.

I don't think we can change line 45 because we need the sentence ID for toggling the correct list selection box.
And I think we don't need jQuery for reading the data attribute.

Reply:
> I don't think we can change line 45

True.

> And I think we don't need jQuery for reading the data attribute.

Yes, I think we can go without jQuery. It could be an issue for people who use Internet Explorer with a version below 11 but that's almost no one among our visitors and that's probably no one among our members who use the list feature.

https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes